src/functions/secrets.ts

```diff
@@ -21,12 +23,25 @@ import { logWarning } from "../utils";
 import { promptOnce } from "../prompt";
 import { validateKey } from "./env";
 import { logger } from "../logger";
-import { functionsOrigin } from "../api";
 import { assertExhaustive } from "../functional";
 import { needProjectId } from "../projectUtils";
 
 const FIREBASE_MANAGED = "firebase-managed";
 
+const gcfV1PollerOptions: Omit<poller.OperationPollerOptions, "operationResourceName"> = {
```

Comment:
*nit: we have the same poller options specified in `fabricator.ts`, should we keep them all in the same place?

Reply:
For some crazy reason... I have to make a copy here other wise some of the value in the options are empty. I wish I knew what was going on... but not going to bother for now. Adding a comment.